plugins/google_directory/plugin.spec.yaml

```diff
@@ -4,7 +4,8 @@ products: [insightconnect]
 name: google_directory
 title: Google Directory
 description: Uses the Google Directory plugin to manage members, users, and user aliases
-version: 2.0.1
+version: 2.0.2
+supported_versions: ["Google Directory API v1 2022-03-29T12:00:00Z"]
```

Comment:
It seems that [Google Directory API Revision History](https://developers.google.com/admin-sdk/directory/v1/revision-history) is not kept up to date as the last version is August 11, 2014, but I can see new methods added and modifications made to the documentations since then - I assume that's why you've added a timestamp.